scripts/lib/install

```diff
@@ -1,61 +1,81 @@
 #!/usr/bin/env bash
 set -xe
 
+# Specify options for apt
+APT_OPTIONS="${APT_OPTIONS:-}"
+# Install additional packages using apt
+ADDITIONAL_PACKAGES=${ADDITIONAL_PACKAGES:-}
+# Call the default type "voyager", for docker it's "dockervoyager"
+DEPLOYMENT_TYPE="${DEPLOYMENT_TYPE:-voyager}"
+# Can be for example "dockervoyager" or voyager
+VOYAGER_CLASS="${VOYAGER_CLASS:-voyager}"
```

Comment:
@timabbott I renamed the variable, but I think it is better to just "call" the main classes (`zulip::voyager` or `zulip::dockervoyager`) here.